components/01-atoms/lists/_list.scss

```diff
@@ -0,0 +1,39 @@
+@mixin reset {
+  list-style: none;
+  margin: 0;
+  padding: 0;
+}
+
+ul {
+  @include reset;
+
+  list-style-type: disc;
+
+  ul {
+    list-style-type: circle;
+
+    ul {
+      list-style-type: square;
+    }
+  }
+}
+
+ol {
+  @include reset;
+
+  list-style-type: decimal;
+
+  ol {
+    list-style-type: lower-alpha;
+
+    ol {
+      list-style-type: lower-roman;
```

Comment:
For the `ol` and `ul`, I don't think we should be making these type of decisions for `list-style-type` - seems like it's very context dependent. 

If this is for an example, maybe move that kind of code into the `stories` file? If not, feel free to let me know the reasoning.

Reply:
I agree with you here. Removed.